rpc/legacyrpc/methods.go

```diff
@@ -1372,13 +1372,14 @@ func makeOutputs(pairs map[string]btcutil.Amount, chainParams *chaincfg.Params)
 // It returns the transaction hash in string format upon success
 // All errors are returned in btcjson.RPCError format
 func sendPairs(w *wallet.Wallet, amounts map[string]btcutil.Amount,
-	account uint32, minconf int32, feeSatPerKb btcutil.Amount) (string, error) {
+	keyScope waddrmgr.KeyScope, account uint32, minconf int32,
```

Comment:
If we already have the account here (which is the internal auto incremented (?) account), then isn't that sufficient? Given that we can look up that account, and then use the keyscopes it defines.

Reply:
Or is there a case where you'd want to force an account to use a different keyscope than what we store with it?

Reply:
Account numbers are not guaranteed to be consistent throughout the different key scopes except for the default account. The key scope requirement here along with the internal account number allow us to uniquely identify an account.